test/DotNetEnv.Tests/ParserTests.cs

```diff
@@ -241,6 +241,23 @@ public void ParseUnquotedValue ()
             Assert.Throws<ParseException>(() => Parsers.UnquotedValue.End().Parse("0\n1"));
             Assert.Throws<ParseException>(() => Parsers.UnquotedValue.End().Parse("'"));
 
+            Assert.Equal("0", Parsers.UnquotedValue.Parse("0\n1").Value);   // value ends on linebreak
+
+            // leading singlequotes/doublequotes/whitespaces/tabs are not allowed
+            Assert.Throws<ParseException>(() => Parsers.UnquotedValue.Parse("'"));
+            Assert.Throws<ParseException>(() => Parsers.UnquotedValue.Parse("\""));
+            Assert.Throws<ParseException>(() => Parsers.UnquotedValue.Parse(" "));
+            Assert.Throws<ParseException>(() => Parsers.UnquotedValue.Parse("\t"));
+
+            Assert.Equal("", Parsers.UnquotedValue.Parse("#").Value); // no value, empty comment
+            Assert.Equal("", Parsers.UnquotedValue.Parse("#commentOnly").Value);
+
+            // prevent quotationChars inside unquoted values
+            Assert.Throws<ParseException>(() => Parsers.Assignment.End().Parse("EV_DNE=a'b'c"));
```

Comment:
These tests are in the wrong function -- this is `ParseUnquotedValue` and should only have tests for `Parsers.UnquotedValue` -- not anything for `Parses.Assignment` as you added here